integration-cli/events_utils.go

```diff
@@ -130,7 +130,7 @@ func matchEventLine(id, eventType string, actions map[string]chan bool) eventMat
 
 		if matchIDAndEventType(matches, id, eventType) {
 			if ch, ok := actions[matches["action"]]; ok {
-				close(ch)
+				ch <- true
```

Comment:
That is still racey too, unfortunately, since both can hit a select like that at the same time.